### PR TITLE
Add module hr_employee_no_welcome

### PR DESCRIPTION
**Pull request by ghost**
This module disables the welcome message when creating an employee from an applicant

### CODE REVIEW

File: hr_employee_no_welcome/README.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+=============
+
+Nothing to do. Only install the module.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to ...
```
Comment (pedrobaeza) [inline hr_employee_no_welcome/README.rst]: Remove this

File: hr_employee_no_welcome/README.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/116/8.0
+
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
```
Comment (pedrobaeza) [inline hr_employee_no_welcome/README.rst]: And this

**Comment by pedrobaeza** [comment]
Minor thing, but :+1: 

It can be very annoying to have this message.

**Comment by ghost** [comment]
@pedrobaeza yep, this message is very annoying for everyone I think. Thanks for the review.

File: hr_employee_no_welcome/models/hr_employee.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+
+class HrEmployee(models.Model):
+
+    _inherit = 'hr.employee'
+
+    @api.model
+    def create(self, vals):
+        self = self.with_context(mail_broadcast=False)
+        return super(HrEmployee, self).create(vals)
```
Comment (moylop260) [inline hr_employee_no_welcome/models/hr_employee.py]: What about use `return super(HrEmployee, self.with_context(mail_broadcast=False)).create(vals)`?
Example: [account_invoice.py#L341](https://github.com/odoo/odoo/blob/ca84c0743779e15fc95fcc24b9c5a93c1858e3a8/addons/account/models/account_invoice.py#L341)

**Comment by moylop260** [comment]
Minor comment 👍

**Comment by jbchouinard** [comment]
:+1:

**Pull request merged**